tests/softreboot_test.go

```diff
@@ -51,6 +51,17 @@ func WaitForVMIRebooted(vmi *v1.VirtualMachineInstance, login func(vmi *v1.Virtu
 	}, 300)).To(Succeed(), "expected reboot record")
 }
 
+func withoutACPI() libvmi.Option {
+	return func(vmi *v1.VirtualMachineInstance) {
+		acpiEnabled := false
+		vmi.Spec.Domain.Features = &v1.Features{
+			ACPI: v1.FeatureState{Enabled: &acpiEnabled},
+		}
+	}
+}
```

Comment:
Would you elaborate? You'd like to "hide it" as an implementation detail? To me it seemed like a useful function; I almost considered promoting it into `libvmi`. Do you have basic principles about in-file ordering of functions, consts etc?

Reply:
> Would you elaborate? You'd like to "hide it" as an implementation detail? To me it seemed like a useful function; I almost considered promoting it into `libvmi`.

No, it doesn't deserve to be hidden away. I agree with you it would make sense to have it in `libvmi`.

> Do you have basic principles about in-file ordering of functions, consts etc?

Yes. I just prefer to focus on the test, and read top to bottom (hence my request to have the `Describe` block first).

Consts are the exception to this ... I'd rather have the consts before the `Describe` block - but I can't reason much about this point; I guess it's historical / something that grew from an habit to culture.